referenceframe/input.go

```diff
@@ -99,19 +99,36 @@ func interpolateInputs(from, to []Input, by float64) []Input {
 	return newVals
 }
 
+// FrameSystemInputs is an alias for a mapping of frame names to slices of Inputs.
+type FrameSystemInputs map[string][]Input
+
 // GetFrameInputs looks through the inputMap and returns a slice of Inputs corresponding to the given frame.
-func GetFrameInputs(frame Frame, inputMap map[string][]Input) ([]Input, error) {
+func (inputs FrameSystemInputs) GetFrameInputs(frame Frame) ([]Input, error) {
```

Comment:
I wonder if functions like this should take a string rather than an entire frame?